bin/migrate-dev-db

```diff
@@ -5,10 +5,11 @@ set -e
 TYPES=src/database/types.ts
 
 npx kysely migrate:latest
-npx kysely seed:run
 
 npx kysely-codegen --camel-case --dialect postgres --out-file $TYPES
 
+npx kysely seed:run
```

Comment:
just out of curiosity wondering why the order of operations changed

Reply:
because the seed files had references to changed columns and it failed because the types file wasn't updated yet, I should add a comment to that effect in the script